rnote-ui/src/canvaswrapper.rs

```diff
@@ -534,6 +551,16 @@ impl RnCanvasWrapper {
     pub(crate) fn set_show_scrollbars(&self, show_scrollbars: bool) {
         self.set_property("show-scrollbars", show_scrollbars.to_value());
     }
+    #[allow(unused)]
+    pub(crate) fn block_pinch_zoom(&self) -> bool {
+        self.property::<bool>("block-pinch-zoom")
+    }
+
+    #[allow(unused)]
+    pub(crate) fn set_block_pinch_zoom(&self, block_pinch_zoom: bool) {
+        self.set_property("block-pinch-zoom", block_pinch_zoom);
+        self.canvas_zoom_gesture_enable(!block_pinch_zoom);
```

Comment:
This causes conflicts with touch drawing, because that toggles the zoom gesture too. One example would be: touch drawing on & block zoom on, disable touch drawing -> gesture is enabled even though block zoom is on.

Reply:
Should be solved now